templates/user/edit-profile.html

```diff
@@ -267,6 +267,12 @@
 {% block body %}
     <div class="centered-form">
         <form id="edit-form" action="" method="post" class="form-area">
+            {% if request.profile.is_peg %}
+                <div class="alert alert-info">
+                    This account was migrated from WCIPEG.
+                    Click <a href="{{ url('wcipeg_merge_request') }}">here</a> to merge with a DMOJ account.
+                </div>
+            {% endif %}
             {% if form.errors %}
```

Comment:
```suggestion
            {% if form.non_field_errors() %}
```

Reply:
Are you sure this is right? This section isn't edited by me and if this is a patch it may conflict with future changes.